tests/test_transport_network.py

```diff
@@ -76,3 +79,27 @@ def test_street_layer_java_object(self, transport_network):
     def test_timezone(self, transport_network, gtfs_timezone_helsinki):
         assert isinstance(transport_network.timezone, java.time.ZoneId)
         assert transport_network.timezone.toString() == gtfs_timezone_helsinki
+
+    def test_cache_directory(self, transport_network_files_tuple):
+        transport_network = r5py.TransportNetwork(*transport_network_files_tuple)
+        cache_dir = transport_network._cache_directory
+        assert cache_dir.is_dir()
+        assert len(list(cache_dir.glob("*"))) > 0  # files have been copied/linked to cache
+        del transport_network  # not sure whether this works as expected
```

Comment:
The tests pass, so this seems to work? So could maybe remove the comment?

Reply:
true, the last comment is superfluous, removing it

Reply:
In fact, I added a comment to the next line to explain what the two lines combined try to test